### ISSUE TITLE
container style

### ISSUE THREAD

**Issue by bergran** [kutlugsahin/ngx-smooth-dnd#5]
Hello mate, at first i want to tell you that you did a great work!
I have a little problem with `smooth-dnd-container`. I would like to put his own style without parent div that style it. I say this because my container is not translating on `orientation vertical` good, i dont know if this is cause i'm using Angular material and styles from your library and AM has conflict.

![image](https://user-images.githubusercontent.com/30236379/39487127-79b7af64-4d7e-11e8-9f09-120491b18baf.png)
![image](https://user-images.githubusercontent.com/30236379/39487134-7e1aa598-4d7e-11e8-9153-fadb5a030054.png)

**Comment by kutlugsahin** [comment]
If you can provide a repo for this issue I can check it. It's hard to tell what causing the problem, but probably it a css issue.

**Comment by kutlugsahin** [comment]
I have check the gist but all seems fine. Can you describe the problem a bit detailed. I still can not understand what is not working. Is it ghost item ? or the draggables in container does not reordered properly when you drag over them ?

**Comment by bergran** [comment]
Yep, it's that you said, it dont reorder on vertical orientation but yes on horizontal. You know why?
